test/unit/application_validator_test.rb

```diff
@@ -67,7 +49,6 @@ class ApplicationValidatorTest < Minitest::Test
 
       config_path = "test/fixtures/skeleton/packwerk.yml"
       configs = YAML.load_file(config_path)
-      configs["inflections_file"] = "custom_inflections.yml"
```

Comment:
We don't need this line anymore as it's [specified directly](https://github.com/Shopify/packwerk/pull/53/files#diff-3f56f44fb50daf4959f7aa276e090b22307338992ad49247e9540b96151e456eR12) in the file.